apps/web/src/components/milestone-logs/recruitment/OfferAcceptedBanner.tsx

```diff
@@ -0,0 +1,37 @@
+import { useApplicantContext } from '../../applicant/ApplicantContext';
+import hiredIndIcon from '@assets/img/hired_indicator.svg';
+import hiredIndCheckmark from '@assets/img/hired_checkmark_banner.svg';
+import { isHired } from '@ien/common';
+
+export const OfferAcceptedBanner: React.FC = () => {
+  const { applicant, offerWithDiffHa } = useApplicantContext();
+
+  const defaultStyle = `flex items-center h-14 mb-5 font-bold rounded`;
+
+  // check for an accepted offer within the same HA as current user
+  const acceptedJobOfferWithSameHa = (): boolean | undefined => {
```

Comment:
it seems inconsistent that `offerWithDiffHa` is set in `ApplicantConext` but `offerWithSameHa` is calculated here.
How about letting context expose `hiredHa` and comparing it with user's ha here?
Fundamentally, it is caused by making ApplicantContext depend on AuthContext. I am not sure it is good or bad. Anyway, I like to put those two calculations together.

Reply:
yup, good point, I had it set up this way when we originally required the full info of the job for a modal, but will redo this chunk and just expose HA instead